src/ansys/dpf/core/data_sources.py

```diff
@@ -376,6 +566,68 @@ def result_files(self):
         ----------
         list
             List of result files.
+
+        Notes
+        -----
+        - If you use the :func:`set_result_file_path() <ansys.dpf.core.data_sources.DataSources.set_result_file_path>`
+        function, it will return only the file path given as an argument to this function.
+
+        Examples
+        --------
+        >>> from ansys.dpf import core as dpf
+        >>>
+        >>> # Create the DataSources object
+        >>> my_data_sources = dpf.DataSources()
+        >>> # Define the path where the main result file can be found
+        >>> my_data_sources.set_result_file_path(filepath='/tmp/file.cas', key='cas')
+        >>> # Add the additional result file to the DataSources object
+        >>> my_data_sources.add_file_path(filepath='/tmp/ds.dat', key='dat')
+        >>> # Get the path to the main result file
+        >>> my_data_sources.result_files
+        ['/tmp/file.cas']
+
+        - If you added an upstream result file, it is not listed in the main ``DataSources`` object. You have to
+        check directly in the ``DataSources`` object created to define the upstream data.
+
+        Examples
```

Comment:
@luisaFelixSalles maybe the second `Examples` section title is the source of error